assets/questions/reverseWords.js

```diff
@@ -88,6 +96,23 @@ globalData.questions['reverseWords'] = {  // eslint-disable-line dot-notation
           "last reversed word."
         ].join('')
       ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseString',
+      messages: [
+        "Read the instructions closely. Did you overlook something important?",
+        [
+          "It looks like you're reversing the entire string instead of each word. "
+        ].join('')
+      ]
+    }, {
+      buggyFunctionName: 'AuxiliaryCode.reverseWordOrder',
+      messages: [
+        "Are you sure you're interpreting the instructions correctly?",
```

Comment:
Mmm, that's a fair point, I agree. Getting the student to understand the question (if they don't already) requires some nuance. I think, though, that it might be worth doing -- "Check the instructions again. If you run your code on {{SAMPLE_INPUT}}, you return {{BUGGY_OUTPUT}}, not {{SAMPLE_OUTPUT}}.", in that case.